airflow/providers/google/cloud/hooks/datafusion.py

```diff
@@ -339,6 +341,8 @@ def delete_pipeline(
         response = self._cdap_request(url=url, method="DELETE", body=None)
         if response.status != 200:
             raise AirflowException(f"Deleting a pipeline failed with code {response.status}")
+        if response.data is None:
+            raise AirflowException("Deleting a pipeline failed because of empty data in received response")
```

Comment:
Update the message: `Empty response received. Please, check for possible root causes of this behavior either in DAG code or on Cloud Datafusion side`

Reply:
Yes, this message sounds more appropriate :)